hack/sync.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#
+# This script sync CRDs from upstream prometheus-operator helm chart
+
+version=${VERSION:-master}
+tmp="/tmp/helm-charts-$version"
+upstream=https://github.com/prometheus-community/helm-charts
+
+
+function main() {
+git clone --depth 1  --branch "kube-prometheus-stack-${version}" "$upstream" "$tmp"
+cp -rf "$tmp/charts/kube-prometheus-stack/crds/" ./helm/prometheus-operator-crd/templates/
```

Comment:
If some files have been deleted in the new version, they will be kept.
I think we should `rm` the previous files (`./helm/prometheus-operator-crd/templates/`)